src/screens/OtherProfile/index.js

```diff
@@ -23,7 +24,6 @@ import BioAndChat from './elements/BioAndChat';
 import BlockIcon from '../../assets/icons/images/block-blue.svg';
 import BlockProfile from '../../components/Blocking/BlockProfile';
 import BottomSheetBio from '../ProfileScreen/elements/BottomSheetBio';
-import EnveloveBlueIcon from '../../assets/icons/images/envelove-blue.svg';
 import GlobalButton from '../../components/Button/GlobalButton';
 import ProfileHeader from '../ProfileScreen/elements/ProfileHeader';
 import ProfileTiktokScroll from '../ProfileScreen/elements/ProfileTiktokScroll';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `EnveloveBlueIcon` has been removed. Make sure that this icon is no longer being used in the component or replaced with a new one.
```diff
- import EnveloveBlueIcon from '../../assets/icons/images/envelove-blue.svg';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->